keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -324,15 +396,21 @@ Contains a unique identifier for the containing cluster.
 
 ##### Uniqueness
 
-*   The identifier **must** be unique within the ClusterSet to which its cluster belongs for the duration of the cluster’s membership.
-*   The identifier **should** be unique beyond the ClusterSet within the scope of expected use.
-*   The identifier **may** be globally unique beyond the scope of its ClusterSet.
-*   The identifier **may** be unique beyond the span of its cluster’s membership and lifetime.
+*   The identifier **must** be unique within the ClusterSet to which its cluster
+    belongs for the duration of the cluster’s membership.
+*   The identifier **should** be unique beyond the ClusterSet within the scope
```

Comment:
Do we need this one?  It's so vague and wishy washy, I find myself trying to figure out what value it offers

Reply:
IMO the value here is to consider future growth and the benefits of global uniqueness vs point-in-time scope uniqueness. It is wishy washy because it's unenforceable. I'm not sure it needs to be codified

Reply:
Move to strike it, then.